### ISSUE TITLE
Issue: CSV File Not Displaying Data

### ISSUE THREAD

**Issue by aixart12** [gosom/google-maps-scraper#81]
After cloning the repository and navigating to the project directory, I started the web application using the following command:

```
go run main.go -web
```

The web app successfully starts at `localhost:8080`. However, despite multiple attempts and debugging sessions, I am unable to resolve an issue where the CSV file shows no data. The app doesn't throw any errors, and both the CSV file and the SQLite database are created successfully.

**Steps to Reproduce:**
1. Clone the repository.
2. Navigate to the project directory.
3. Run `go run main.go -web`.
4. Check the web app and observe that the CSV file has been created but contains no data.

**Expected Behavior:**
The CSV file should contain data as expected when interacting with the web app.

**Actual Behavior:**
The CSV file is created, but it is empty. No errors are shown in the logs or console output.

**Screenshots:**
- Web app running on `localhost:8080`.
- Screenshot of the CSV and SQLite database files showing successful creation.

**Environment:**
- Go version: `go1.23.1 linux/amd64`
- Running on Windows WSL

**Additional Information:**
- System OS: Windows WSL
- Database: SQLite

Any help in resolving this issue would be greatly appreciated!

![image](https://github.com/user-attachments/assets/16605609-db40-4613-a3af-f59ed7e78923)
![image](https://github.com/user-attachments/assets/275951a1-c0e6-4b97-b3c0-45ad173ae451)
![image](https://github.com/user-attachments/assets/e5a20cf0-a5b5-4f05-a5a0-bbef22224d9f)

**Comment by gosom** [comment]
thanks for the report. 

I think it's because I set the exit on inactiviy to 20s and the crawler takes some time to start. 

I am increasing to 3m and releasing a new version

**Comment by gosom** [comment]
@aixart12 a new version is released with the fix.

Will you mind to check again and close the issue if it's resolved also in windows?

**Comment by aixart12** [comment]
@gosom  , I'm running the app locally on Windows using WSL (Ubuntu 24.04), but when I try to run it again, I'm still encountering the same issue. Is there something I might be missing to run the web app properly?

Also, I downloaded the .exe file from GitHub, but it gets stuck on a black screen.
![image](https://github.com/user-attachments/assets/0eb173bf-f3ec-46e7-bef2-3ff623b5f548)

**Comment by gosom** [comment]
how you started the app?

**Comment by aixart12** [comment]
go run main.go -web

**Comment by gosom** [comment]
no from the exe file, I assume similarly right?

**Comment by aixart12** [comment]
to run .exe file I am just running it like any other file , jut double click it .

**Comment by gosom** [comment]
this won't work and the behavior is expected. 
you need to start from the command line and use the -web flag. 

But, I believe probably it should start by default in web mode. 

can you try that and let me know?

**Comment by aixart12** [comment]
@gosom, it worked with the command:
```
.\google_maps_scraper-1.4.2-windows-amd64.exe -web
```
This time, I also got results in the CSV file.

However, I noticed that if I cancel the app during the scraping process, the job status remains as 'working' even though no job is actually running.


And Also Could you help me understand why I'm unable to run the app through the code?

![image](https://github.com/user-attachments/assets/e70a227c-905a-4efe-a110-dc1fa861360f)

**Comment by gosom** [comment]
> However, I noticed that if I cancel the app during the scraping process, the job status remains as 'working' even though no job is actually running.


Yes this is expected behavior. There is no way to restart or remove at the moment. 

this should work through the code. However I don't know via WSL. 

Have you tried the docker container?

**Comment by gosom** [comment]
@aixart12 is this still an issue?

**Comment by Distrect** [comment]
I have the same issue. I am using macos and created .app file for it. When i do query it retruns as empty csv. I also tried cli and web ui but same result

**Comment by Jessi0309** [comment]
<img width="1719" height="1038" alt="Image" src="https://github.com/user-attachments/assets/2cc00256-60fc-4e0a-a63f-8abe1b5338a3" />same question， can‘t do anything......
